src/apps/startups/models.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils import timezone
+
+from apps.users.models import Founder, User
+
+
+class Project(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    project_name = models.CharField("Project Name", max_length=64)
+    PROJECT_TYPE = (("P", "Project"), ("I", "Idea"))
+
+    type = models.CharField(
+        max_length=1,
+        choices=PROJECT_TYPE,
+        blank=True,
+        default="S",
+        help_text="Project type",
+    )
+    created_at = models.DateTimeField()
+    description = models.TextField("Bio", max_length=425, blank=True, null=True)
```

Comment:
```suggestion
    description = models.TextField("Bio", max_length=425, blank=True)
```
Not necessary as blank is set

Reply:
Fixed in 0c4d9946c144c957d6e3b0a064ce36d18b46d83a